WcaOnRails/app/webpacker/components/Panel/Leader/GroupsManager.jsx

```diff
@@ -54,7 +54,7 @@ export default function GroupsManager({ loggedInUserId }) {
   }, [roles]);
 
   if (!loading && roles.length === 0) return <p>You are not a leader of any groups.</p>;
-  if (loading || selectedGroupId === null) return <Loading />;
+  if (loading || selectedGroupId == null) return <Loading />;
```

Comment:
Earlier, the condition was satisfied only when selectedGroupId was null, but undefined should also be handled, so changed `===` to `==`.

Reply:
Abusing the fact that "accidentally" null and undefined behave the same way when using `==` is not a good approach.
I would favor either
1. Using `!selectedGroupId`
2. Refactoring so that `selectedGroupId` only uses null OR undefined, but cannot become both.
(with a strong preference for option (2), but giving option (1) as a backup if the other is too hard)

Reply:
Okay, will go with 2, I just checked and there is no case where `selectedGroupId` is `null`, so I changed it to `=== undefined`.
1st option is not possible because `selectedGroupId` can be 0 as well.